packages/firestore/test/unit/specs/remote_store_spec.test.ts

```diff
@@ -77,6 +77,7 @@ describeSpec('Remote store:', [], () => {
         // Close before we get an ack, this should reset our pending
         // target counts.
         .watchStreamCloses(Code.UNAVAILABLE)
+        .expectEvents(query, { fromCache: true })
         // This should work now.
```

Comment:
I don't understand what this comment means anymore now that we've raised the event above.